frontend/packages/citizen-frontend/src/util.ts

```diff
@@ -0,0 +1,15 @@
+// SPDX-FileCopyrightText: 2017-2021 City of Espoo
+//
+// SPDX-License-Identifier: LGPL-2.1-or-later
+
+import { format } from 'date-fns'
+
+export const DATE_FORMAT_DEFAULT = 'dd.MM.yyyy'
+export const DATE_FORMAT_DATE_TIME = 'dd.MM.yyyy HH:mm'
```

Comment:
Since we have `LocalDate` for dates without a time component would making `'dd.MM.yyyy HH:mm'` the default make more sense?